rest_framework_simplejwt/serializers.py

```diff
@@ -123,6 +129,16 @@ def validate(self, attrs: Dict[str, Any]) -> Dict[str, str]:
             refresh.set_exp()
             refresh.set_iat()
 
+            auth = JWTAuthentication()
+            user = auth.get_user(validated_token=refresh)
+            OutstandingToken.objects.create(
+                user=user,
+                jti=refresh[api_settings.JTI_CLAIM],
+                token=str(refresh),
```

Comment:
```suggestion
                token=data["refresh"],
```
This part where we encode refresh token is used twice in this code. One is here on line number 137 and another is on line:142 

Maybe its bad idea to repeat this code, because in future refactor to change or update this value we might need to look into  2 places. 

My suggestion will be  move 142 on top then use `token = data["refresh" ]`

Hope this gets merged soon :smiley: